universal-application-tool-0.0.1/app/assets/javascripts/preview.ts

```diff
@@ -15,14 +15,16 @@ class PreviewController {
   static readonly DEFAULT_QUESTION_HELP_TEXT = "Sample question help text";
   static readonly DEFAULT_ENTITY_TYPE = "Sample repeated entity type";
 
+  static readonly THIS_REGEX = /(\$this(?:\.parent)*)/g;

```

Comment:
maybe add a brief comment explaining the function of this regex?

Reply:
added